docs/configuration.md

```diff
@@ -0,0 +1,257 @@
+# Configuration
```

Comment:
I think we need a TLDR at the beginning of this file explaining how to make a config with one JWT and a hostname 

```

[credentials]
domain="palantir.mcloud.merckgroup.com"
token_provider.config.jwt="eyJwb..."
```
and one with oauth client_id/client_secret.
And than tell the user afterwards to run `fdt config` to check if all good.